src/host/ft_host/Message_KeyPressTests.cpp

```diff
@@ -86,7 +86,6 @@ class KeyPressTests
         expectedRecord.Event.KeyEvent.uChar.UnicodeChar = 0x0;
         expectedRecord.Event.KeyEvent.bKeyDown = true;
         expectedRecord.Event.KeyEvent.dwControlKeyState = ENHANCED_KEY;
-        expectedRecord.Event.KeyEvent.dwControlKeyState |= (GetKeyState(VK_NUMLOCK) & KEY_STATE_TOGGLED) ? NUMLOCK_ON : 0;

```

Comment:
This was quite puzzling. On line 78 above we call `TurnOffModifierKeys(hwnd)` and then assume that conhost still thinks that the numlock is set? This test should've always failed on every keyboard where numlock was on.